RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel Serialization release notes
 
+## 2.1 (dev)
+
+* Added support for facets
+
```

Comment:
Actually, this only adds support for the `LanguageFallbackInfo` facet and removes support for `TermFallback`.